### PR TITLE
Update configmap-template.yaml

### PR DESCRIPTION
**Pull request by hemajv**
Added the Prometheus pushgateway host and port variables

Depends-On: #89 
Depends-On: #90 
Depends-On: #91

### CODE REVIEW

File: openshift/configmap-template.yaml [hunk 1]
```diff
@@ -117,3 +119,15 @@ parameters:
   name: RSYSLOG_PORT
   required: true
   value: '10514'
+  
+- displayName: PROMETHEUS_PUSHGATEWAY_HOST
+  description: A host to start the pushgateway server.
```
Comment (fridex) [inline openshift/configmap-template.yaml]: We are not starting the pushgateway. The description should state that the metrics will be pushed to this gateway, if the gateway configuration is provided.

Comment (hemajv) [reply openshift/configmap-template.yaml]: Okay, got it!

**Comment by goern** [comment]
recheck

**Comment by goern** [comment]
recheck

**Pull request closed without merging**